### DISCUSSION TITLE
What is the "proper" way to share my data with others ?

### DISCUSSION THREAD

**Discussion by qiulang** [mockoon/mockoon#433] [Help] [answered]
What is the "proper" way to share my data with others ?
Now I export the json file, add to git and ask others to import it. It seems the only way to share with others.

Thanks

Qiulang

**Comment by 255kb** [comment, marked as answer]
Hi @qiulang Indeed, this is currently the only way to share data with others. One improvement is on the roadmap to allow you to choose the storage folder for each environment to easier the sharing: https://github.com/mockoon/mockoon/issues/16
You could also try to set up a git repository inside Mockoon's data storage folder `~/{user_data_folder}/Mockoon/storage` as a temporary solution.

**Reply by qiulang** [reply]
Hi thanks for replying my question. So I would like to make another suggestion, export yaml format, so it is more human readable. 
For export OpenAPI json format I feel it has bugs because the content is basically empty and when I  import it again it confirms the problem.

**Reply by 255kb** [reply]
Please do not hesitate to open a new issue if you find a reproductible bug. OpenAPI export should work, if it's not maybe you encountered a bug.

**Reply by qiulang** [reply]
Hi I opened an issue https://github.com/mockoon/mockoon/issues/437  for that.
Thanks

**Reply by 255kb** [reply]
Thank tou @qiulang Now I see what you meant. In fact, it's not a bug. This has not been implemented yet. There is an opened issue already for this: https://github.com/mockoon/mockoon/issues/352
